resources/views/pages/import.blade.php

```diff
@@ -2,23 +2,37 @@
 
 @section('main_content')
 
-@if (in_array(request()->input('type'), \Chompy\ImportType::all()))
+@if ($type === \Chompy\ImportType::$rockTheVote)
     <div>
+        <h1>Rock The Vote</h1>
+        <p>Creates or updates users and posts from CSV.</p>
+        <h4>Users</h4>
+        <dl>
+            <dt>Source:</dt><dd>{{ config('services.northstar.client_credentials.client_id') }}</dd>
+            <dt>Reset email enabled</dt><dd>{{ $resetConfig['enabled'] ? 'true' : 'false'}}</dd>
```

Comment:
Does this have to be a ternary since `$resetConfig['enabled']` is a boolean? Can we just return `$resetConfig['enabled']`?

Reply:
Yeah, when I tried printing the raw boolean value - it rendered as an empty string when `false`.

Reply:
oh interesting!